sdcm/cluster_aws.py

```diff
@@ -593,6 +634,18 @@ def _instance_wait_safe(self, instance_method: Callable[P, R], *args: P.args, **
     def _wait_public_ip(self):
         ec2_instance_wait_public_ip(self._instance)
 
+    @cached_property
+    def cql_address(self):
+        address = self.scylla_network_configuration.test_communication \
```

Comment:
I think this logic here warrants a comment explaining why 
"""
cause of test is doing CQL communication with the python-driver
"""

at some point we might want to have separate attributes, for the test, and for loader for example.

cause this would cause also loader to work with public address to do CQL commands, since it's only for local development, it not that critical, but we might want to align it at some point (not now)